### PR TITLE
travis: use container based build, PHP 7 added

### PR DESCRIPTION
**Pull request by TomasVotruba**
Also `--dev` is by default over a year now.

### CODE REVIEW

**Comment by stof** [comment]
the installation of ``riak-beta`` should be disabled on PHP 7 (as done on HHVM) as it is not yet updated for PHP 7.

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +23,7 @@ before_install:
     - sh -c "if [ $TRAVIS_PHP_VERSION != 'hhvm' ]; then phpenv config-add ./tests/travis/php.ini; fi"
 
 install:
-    - composer --prefer-source --dev install
+    - composer --prefer-source install
```
Comment (GrahamCampbell) [inline .travis.yml]: more normal to add the flags after the command name: `composer install --prefer-source`.

**Comment by TomasVotruba** [comment]
What is missing here? Related only to this PR's intention.

**Comment by Ocramius** [comment]
@TomasVotruba see build errors.

**Comment by TomasVotruba** [comment]
Fixed now

**Comment by Ocramius** [comment]
@TomasVotruba thanks!

**Comment by TomasVotruba** [comment]
Thank you!

**Pull request merged**